yarn-project/circuit-types/src/interfaces/prover-coordination.ts

```diff
@@ -16,4 +16,9 @@ export interface ProverCoordination {
    * @param quote - The quote to store
    */
   addEpochProofQuote(quote: EpochProofQuote): Promise<void>;
+
```

Comment:
Adding stop here so that we can call stop on the p2p client in the stop of the prover node without needing to separately provide a reference to the p2p client